examples/basic/audio-concat-transcoder.rb

```diff
@@ -0,0 +1,41 @@
+class AudioConcatTranscoder < MediaTranscoder
+  require 'transloadit'
+  require './media-transcoder'
+
+  # in this example a file is encoded as an mp3, id3 tags are added, and it is stored in s3
+  def transcode!(files)
+    concat = transloadit_client.step('concat', '/audio/concat', {
+      ffmpeg_stack: 'v2.2.3',
```

Comment:
@tim-kos @kvz you know best, but is this a required param? Or can we leave that out to ease that example a little bit?